app/scripts/modules/core/instance/instances.component.ts

```diff
@@ -121,6 +136,16 @@ export class InstancesComponent implements IComponentOptions {
   public controller: any = InstancesController;
 }
 
+export class InstancesWrapperComponent implements IComponentOptions {
+  public bindings: any = {
+    instances: '<',
+    highlight: '<'
+  };
+  public controller: any = InstancesController;
+  public template = `<instances instances="$ctrl.instances" highlight="$ctrl.highlight"/>`;
```

Comment:
I don't understand this.  If Instances component has one way bindings, why do you need to wrap it?